### PR TITLE
Adding file with things to do in Schaumburg

### PR DESCRIPTION
**Pull request by nkmadhav1**
This pull request introduces file for Schaumburg

### CODE REVIEW

**Comment by jianchengit** [comment]
looks good!

**Comment by qqshannon** [comment]
Looks good! :)

File: test.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Schaumburg
+
+### Places to eat
```
Comment (qqshannon) [inline test.md:3]: -Mitsuwa Marketplace

**Review by qqshannon — COMMENTED** [review]
Some place to eat

**Pull request merged**